src/index.html

```diff
@@ -10,8 +10,137 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="field-group">
+        <legend>Personal information</legend>
+        <label for="surname">Surname:
+          <input class="form-controls"
+          type="text"
+          autocomplete="off"
+          name="surname"
+          minlength="2"
+          maxlength="30"
+          required
+          >
+        </label> <br>
+        <label for="name">Name:
+          <input class="form-controls"
+          type="text"
+          autocomplete="off"
+          name="name"
+          minlength="2"
+          maxlength="30"
+          required>
+        </label> <br>
+        <label for="age">How old are You?
+          <input
+          class="form-controls"
+          type="number"
+          value="12"
+          min="1"
+          max="100"
+          name="age"
+          required>
```

Comment:
Indentation. Please, do this in the rest of the file

```suggestion
          <input
            class="form-controls"
            type="number"
            value="12"
            min="1"
            max="100"
            name="age"
            required
          >
```